framework/src/main/java/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -544,7 +543,20 @@ private void loadBundledAnnotationClasses() {
             // We do not support a resource URL with any other protocols, so create an empty set.
             annotationNames = Collections.emptySet();
         }
-
+        if (annotationNames.isEmpty()) {
+            PackageElement pkgEle = checker.getElementUtils().getPackageElement(packageName);
+            if (pkgEle != null) {
+                for (Element e : pkgEle.getEnclosedElements()) {
+                    if (e.getKind() == ElementKind.ANNOTATION_TYPE) {
+                        // Elements needs to be annotated.
```

Comment:
If this is a justification for the `@SuppressWarnings` annotation, please put it on the same line as an end-of-line comment.